Apps/W1/Shopify/app/src/Invoicing/Codeunits/ShpfyFulfillmentAPI.Codeunit.al

```diff
@@ -0,0 +1,53 @@
+namespace Microsoft.Integration.Shopify;
+
+/// <summary>
+/// Codeunit Shpfy Fulfillment API (ID 30315).
+/// </summary>
+codeunit 30315 "Shpfy Fulfillment API"
+{
+    var
+        ShpfyJsonHelper: Codeunit "Shpfy Json Helper";
+        ShpfyCommunicationMgt: Codeunit "Shpfy Communication Mgt.";
+
+    /// <summary>
+    /// Fulfills shopify orders for each fulfillment id parsed from a completed draft order.
+    /// </summary>
+    /// <param name="JResponse">Json response from a completed draft order</param>
+    /// <param name="ShopCode">Shopify shop code to be used.</param>
+    internal procedure FulfillShopifyOrder(JResponse: JsonToken; ShopCode: Code[20])
```

Comment:
The procedure should take in a FulfillmentOrderList, so that it can be used more generically, and not bound specifically to one single response json.

Parsing of JResponse into Fulfillment Order List should happen in the orchestrator codeunit.

Reply:
Better yet, it should take in just one Fulfillment order id. And again the orchestrator can loop and call the api multiple times.